plugins/logicalaccess/plugins/readers/pcsc/atrparser.cpp

```diff
@@ -464,8 +464,6 @@ std::string ATRParser::check_generic_from_atr() const
                                           "retrieve Category Indicator.");
                 /*unsigned char categoryIndicator = atr[y + x++];	// See
                  * ISO7816-4, 8.2*/
-                x++;
```

Comment:
This break the alignment parsing... Please comment the lines
```
unsigned int x = 0;
EXCEPTION_ASSERT_WITH_LOG(x < historicalBytesLength,
    LibLogicalAccessException,
    "Bad historical buffer. Too short to "
    "retrieve Category Indicator.");
```
and init directly x to 1
`unsigned int x = 1;`